src/common/types/column/column_data_collection.cpp

```diff
@@ -485,19 +485,18 @@ void ColumnDataCopy<string_t>(ColumnDataMetaData &meta_data, const UnifiedVector
 			if (entry.IsInlined()) {
 				continue;
 			}
-			if (heap_size + entry.GetSize() > Storage::BLOCK_SIZE) {
+			if (heap_size + entry.GetSize() > Storage::DEFAULT_BLOCK_SIZE) {
```

Comment:
We could use the allocator's buffer manager here as well

Maybe we can add this method to the `ColumnDataAllocator`:

```cpp

BufferManager &ColumnDataAllocator::GetBufferManager() {
   if (type == ColumnDataAllocatorType::IN_MEMORY_ALLOCATOR) {
        throw InternalException("go away");
   } 
   return *alloc.buffer_manager;
}
```